docs/source/student_tutorial/index.rst

```diff
@@ -0,0 +1,48 @@
+============
+Getting Started
+============
+
+Revolve2 is a library that allows you to use evolutionary computing for modular robot evolution and simulation of those robots.
+Additionally these modular robots can be built physically and subsequently be controlled by the brain of your simulated robots.
+In this tutorial we will go over all steps for you to start researching ASAP.
+
+-------------
+Step 1: Installation
+-------------
+The first step is to install revolve2. For this follow the detailed tutorial from :ref:`Installation Guide<installation/index>`.
+Keep in mind that installation can vary depending on the release you choose.
+
+---------------------
+Step 2: Play with examples
+---------------------
+Depending on your topic of research some different functionality in revolve2 might be interesting to you.
+One good starting point to get a better understanding of the inner workings of revolve2, are the examples found in the repository.
+
+* To understand the mechanics behind the simulations better look at: `simulate_single_robot`
+
+* If you need a refresh on evolutionary algorithms try to understand the following examples:
+
+    * `simple_ea_xor`and `simple_ea_xor_database`
```

Comment:
I would recommend without database, and recommend with database afterwards to understand the concept of a database. in the past the database was really confusing to students